integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,51 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupTeam"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .id + \"/home\""
+            member: (.members | map(.user.username | tostring))
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupProject"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .__team_id + \"/v/b/li/\" + .id "
+            startDate: if .start_date != null then (.start_date | todate) else "946684800" | todate  end  # Unix epoch for 01/01/2000
+            endDate: if .due_date != null then (.due_date | todate) else "946684800" | todate end  # Unix epoch for 01/01/2000
+            totalIssues: .task_count
+          relations:
+            team: .__team_id
+  - kind: issue
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupIssue"'
+          identifier: .id
+          title: .name
+          properties:
+            description: .description
+            url: .url
+            status: .status.status
+            assignee: if (.assignees | length) > 0 then .assignees[0].username else "" end
+            creator: .creator.username | tostring
+            priority: if .priority != null then .priority | tostring else " " end
+            created: if .date_created != null then (.date_created | todate) else "946684800" | todate end  # Unix epoch for 01/01/2000
+            updated: if .date_updated != null then (.date_updated | todate) else "946684800" | todate  end  # Unix epoch for 01/01/2000
```

Comment:
Why are you hardcoding 946684800? If there is no created date then there shouldn't be any value for it

Reply:
I have fixed this, it is now empty when not present